cmake/FindNodejs.cmake

```diff
@@ -53,23 +52,19 @@ find_path(V8_CONFIG_INCLUDE_DIR
 # libnode.so.111, node 19.0.0
 #
 set(nodejs_known_names
-    "node"
-    "libnode.so"
-    "libnode.so.83"
-    "libnode.so.93"
-    "libnode.so.102"
-    "libnode.so.108"
-    "libnode.so.111"
+    "libnode.so" "libnode.dylib"
+    "libnode.so.83" "libnode.83.dylib"
+    "libnode.so.93" "libnode.93.dylib"
+    "libnode.so.102" "libnode.102.dylib"
+    "libnode.so.108" "libnode.108.dylib"
+    "libnode.so.111" "libnode.111.dylib"
 )
 
 find_library(NODEJS_LIBRARY
     NAMES ${nodejs_known_names}
     PATHS ${NODEJS_ROOT_DIR}/lib
     NO_DEFAULT_PATH
 )
-find_library(NODEJS_LIBRARY
```

Comment:
If one doesn't set NODEJS_ROOT_DIR, it should be using this one. Do things continue to work if you add this back?

Reply:
Ahhhhh, I didn't understand why that was there. I would think it should be fine.

Reply:
I'll update PR later.

Reply:
Fixed